environment/src/main/java/com/sequenceiq/environment/parameters/validation/validators/parameter/AzureParameterValidator.java

```diff
@@ -156,8 +158,16 @@ private ValidationResult validateEncryptionParameters(ValidationResultBuilder va
                                 + "but that feature is currently disabled. Get 'CDP_CB_AZURE_DISK_SSE_WITH_CMK' enabled for your account to use SSE with CMK.").
                         build();
             }
+            if (encryptionKeyResourceGroupName == null && USE_MULTIPLE.equals(azureParametersDto.getAzureResourceGroupDto().getResourceGroupUsagePattern())) {
+                LOGGER.info("Invalid request, Neither --encryption-key-resource-group-name nor --resource-group-name is present.");
+                return validationResultBuilder.error(
+                        "To use Server Side Encryption for Azure Managed disks with CMK, Either --encryption-key-resource-group-name or " +
+                                "--resource-group-name should be specified. Please provide --resource-group-name, If encryption key is present in the same " +
+                                "resource group you wish to create the environment in, Else provide --encryption-key-resource-group-name.").
```

Comment:
Nit: make "Else" lowercase. But this sentence may have to be reformulated as per my above comment.